code/modules/mob/living/carbon/human/species/unathi.dm

```diff
@@ -64,11 +64,13 @@
 	..()
 	lash = new
 	lash.Grant(H)
+	H.mind.offstation_role = TRUE
```

Comment:
This sets all **_unathi_** players as offstation roles. Ashlwakers are a subtype of unathi, not the main race.
Consider setting this specifically to the mindm when ashwalkers spawn, so as not to make people who somehow become walkers (changelings, etc.) also unselectable for objectives.